gui/next/src/routes/backgroundJobs/[type]/[id]/+page.svelte

```diff
@@ -145,6 +145,13 @@ code {
       </Code>
     {/if}
 
+    {#if item.partial_name}
+      <h2>Background function name:</h2>
+      <Code>
```

Comment:
How about just `<code>` so it will render it without line numbers etc. as it does with the 'Arguments' section below?

Reply:
Fixed.